jans-config-api/plugins/admin-ui-plugin/src/main/java/io/jans/ca/plugin/adminui/service/webhook/WebhookService.java

```diff
@@ -338,9 +338,11 @@ public void validateWebhookEntry(WebhookEntry webhookEntry) throws ApplicationEx
                 throw new ApplicationException(Response.Status.BAD_REQUEST.getStatusCode(), ErrorResponse.WEBHOOK_CONTENT_TYPE_REQUIRED.getDescription());
             }
         }
-        if (getBlockedUrls().contains(webhookEntry.getUrl())) {
-            log.error(ErrorResponse.WEBHOOK_URL_BLOCKED.getDescription());
-            throw new ApplicationException(Response.Status.BAD_REQUEST.getStatusCode(), ErrorResponse.WEBHOOK_URL_BLOCKED.getDescription());
+        for (String blockedUrl : getBlockedUrls()) {
+            if (webhookEntry.getUrl().contains(blockedUrl)) {
```

Comment:
@433B finally in this commit bug was found. Sounds like test helped to catch it. However one thing that still is there is comparison problem. E.g. blockedUrl=`172.`, means that `152.213.172.23` will be blocked because `172.` is inside. Does it make sense to add pattern like `172.*` where we explicitly specify what should be blocked? E.g. `*172.*` and `172.*` and `*.172` are all different type of blocking.